client/extensions/woocommerce/state/sites/orders/activity-log/selectors.js

```diff
@@ -52,8 +53,16 @@ export const EVENT_TYPES = {
  * @return {boolean} Whether the activity log for a given order has been successfully loaded from the server.
  */
 export const isActivityLogLoaded = ( state, orderId, siteId = getSelectedSiteId( state ) ) => {
-	return areOrderNotesLoaded( state, orderId, siteId ) &&
-		( ! config.isEnabled( 'woocommerce/extension-wcservices' ) || areShippingLabelsLoaded( state, orderId, siteId ) );
+	const notesLoaded = areOrderNotesLoaded( state, orderId, siteId );
+	if ( ! notesLoaded ) {
+		return false;
+	}
+
+	if ( ! config.isEnabled( 'woocommerce/extension-wcservices' ) || areShippingLabelsErrored( state, orderId, siteId ) ) {
```

Comment:
Is there an env where this still isn't enabled? Guessing Desktop, but since this is live in production now do we need to have this config check?